### PR TITLE
adds exam ingest validation tenant admin config section

### PR DESCRIPTION
**Pull request by esotrope**
image

### CODE REVIEW

File: webapp/src/main/webapp/src/app/admin/tenant/model/data/configurable-data-elements.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+export const configurableDataElements = [
+  'FirstName',
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/admin/tenant/model/data/configurable-data-elements.ts]: should this order be respected in the list or should these be sorted alphabetically?

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 2]
```diff
@@ -1527,7 +1527,8 @@
       "datasources": "Data Sources",
       "task": "Task",
       "archive": "Archive",
-      "aggregate": "Aggregate"
+      "aggregate": "Aggregate",
+      "validation": "Exam Ingest Validation"
```
Comment (esotrope) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: @malaffoon does this name make sense?

File: webapp/src/main/webapp/src/app/admin/tenant/model/data/configurable-data-elements.ts (hunk 1 above)
Comment (malaffoon) [reply webapp/src/main/webapp/src/app/admin/tenant/model/data/configurable-data-elements.ts]: Probably alphabetical. There's no "right" order and alphabetical makes it easier to find things.

File: webapp/src/main/webapp/src/assets/i18n/en.json (hunk 2 above)
Comment (malaffoon) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: Sure, although you could probably make a bit more generic, just "Ingest Validation". A couple of the validations are for test packages.

**Pull request merged**